statement.k

```diff
@@ -44,8 +44,108 @@ rule
     <k> #exeStmt(S:SimpleStatement;) => S; ...</k>
     <callStack> .List </callStack>
 
-rule
-    <k> #localFunctionCall(FN:Id, ARGS:Values) => functionCall(FN; ARGS) ... </k>     [structural]
+syntax ContractCellMap
+syntax Map ::= ContractCellMap
+
+
+//**************************************************************************//
+// I was planing to have the following rule for address conversion.
+// However, the behavior of the in_keys() funciton is not what I expected.
+// Thus, I have another work-around, which works, but may not be perfect.
+// Maybe someone can help me to check on this?
+
+/*
+rule [Address-Conversion]:
+    <k> #localFunctionCall(FN:Id, V:Value, .Values) => V ... </k>
+    <contracts> CTS </contracts>
+    requires (FN in_keys(CTS))
+*/
+
+rule [Address-Conversion]:
+    <k> #localFunctionCall(FN:Id, V:Value, .Values) => V ... </k>
```

Comment:
Wrap up the values like `#localFunctionCall(FN:Id, (V:Value, .Values))` to make it so that K knows that `V` and `.Values` are part of the same list instead of separate arguments to `#localFunctionCall`.